jupyterlab_nodeeditor/__init__.py

```diff
@@ -43,4 +43,4 @@ def _load_jupyter_server_extension(server_app):
     OutputSlot,
     SocketCollection,
 )
-from .yggdrasil_support import parse_yggdrasil_yaml
+# from .yggdrasil_support import parse_yggdrasil_yaml
```

Comment:
Why is this commented out?  I thought this was the function you were implementing.

Reply:
I believe `parse_yggdrasil_yaml` was your function before I made mine. Regardless, it's been updated with `model_loader` now.